jenkins/github_helpers.py

```diff
@@ -286,7 +286,6 @@ def verify_upgrade_packages(self, pull_request):
             if not suspicious_reqs and valid_reqs:
                 pull_request.set_labels('Ready to Merge')
                 logger.info("Total valid upgrades are %s", valid_reqs)
-                self._add_comment_about_reqs(pull_request, "Valid upgraded packages", valid_reqs)
```

Comment:
The packages which don't have an issue are already added on PR here: https://github.com/openedx/testeng-ci/blob/a392ccefcfef5941a52c892633a827888ec83405/jenkins/github_helpers.py#L284

Example PR with the duplicate comment: https://github.com/edx/edx-e2e-tests/pull/571